### DISCUSSION TITLE
Welcome to RSE_Asia Discussions!

### DISCUSSION THREAD

**Discussion by SaranjeetKaur** [rse-asia/RSE_Asia#33] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by SaranjeetKaur** [comment]
Okay, here I am adding updates from the Hacktoberfest 2022 Contributing Hours, that we have had so far!

- 3 October 2022 (Monday): @kozo2 started working on the RSEAA unconference experience blog, @SaranjeetKaur updated the README of the repo.
- 4 October 2022 (Tuesday): @kozo2 developed a draft of the blog, @SaranjeetKaur updated the Talks page.
- 5 October 2022 (Wednesday): No contributing hours conducted.
- 6 October 2022 (Thursday): @kozo2 added the blog to the website template, @SaranjeetKaur created the Blog page and added a blog template for @kozo2 Kozo, she also created the Events page and added the RSE Asia Google calendar to it, and also added blog template for @jyoti-bhogal, @annajiat explored the ideas to work on and what all has been happening so far, suggested to take notes daily and hopes to contribute to CoC and template creation.

**Comment by SaranjeetKaur** [comment]
- 7 October 2022 (Friday): @SaranjeetKaur working on updating the news page
(@kozo2, @annajiat - please add your updates here as comments for 7 October 2022)

**Reply by annajiat** [reply]
@Annajiat contemplated the possibility of a CoC committee and a potential ombudsperson swap with another open-source community. Completed the Hacktoberfest registration.

**Reply by SaranjeetKaur** [reply]
just sharing here to keep everyone in the loop, we do have a couple of people who joined the CoC team within the Working Group (CC: @sayantikabanik, @amangoel185)

**Reply by kozo2** [reply]
I completed the Hacktoberfest registration.
I am writing a blog post about my impression of [RSEAA](https://rse-aunz.github.io/2022-Asia-Australia-unconference/).
I will publish the post in two languages, Japanese and English.
After writing the post in Japanese first, I plan to create an English post as an English translation of the Japanese post.
Japanese posts are created in [this branch](https://github.com/kozo2/RSE_Asia/tree/kozo-japanese-blogpost-template).

**Comment by SaranjeetKaur** [comment]
- 10 October 2022 (Monday): @SaranjeetKaur updated the news page and started working on adding all contributors to the repo

**Comment by SaranjeetKaur** [comment]
- 11 October 2022 (Tuesday): @SaranjeetKaur updated the contributors on the repo

**Comment by kozo2** [comment]
- 11 October 2022 (Tuesday): @kozo2 finished writing the Japanese post about RSEAA #32

**Comment by kozo2** [comment]
- 12 October 2022 (Wednesday) @kozo2 started translating the Japanese post to English (but not finished yet).

**Comment by jyoti-bhogal** [comment]
12 October 2022 (Wednesday) @jyoti-bhogal 

1.  Read and understood the Speed Blog template provided by OLS in their github repository
2. Went through the speed blogs written by fellow OLS-5, and OLS-4 mentees
3. Collected text materials, links, and pictures to be included in my speed blog
4. Discussed with @SaranjeetKaur and cleared my certain doubts operating Github forking, pushing commit, and creating pull request.

**Comment by SaranjeetKaur** [comment]
- 12 October 2022 (Wednesday): @SaranjeetKaur explored about how to port rmd to qmd. Links to related blogs are shared below:
1. https://blog.djnavarro.net/posts/2022-04-20_porting-to-quarto/
2. https://www.njtierney.com/post/2022/04/11/rmd-to-qmd/
3. https://karatsidhu.com/posts/make-quarto-website/make-quarto-website.html
4. https://karatsidhu.medium.com/converting-a-distill-website-to-quarto-35f2422d0b53
5. https://quarto.org/docs/websites/

**Comment by jyoti-bhogal** [comment]
18 October 2022 (Tuesday): @jyoti-bhogal 
1. Progressed on writing her blog

**Comment by kozo2** [comment]
18 October 2022 (Tuesday): @kozo2 

1. Progressed on writing https://github.com/rse-asia/RSE_Asia/pull/44 and https://github.com/rse-asia/RSE_Asia/pull/32

**Comment by SaranjeetKaur** [comment]
19 October 2022 (Wednesday): @SaranjeetKaur 
- Supported @jyoti-bhogal with updating her blog and working on some git commit issues
- Gave a walkthrough to @amangoel185 of the things happening so far and had discussions about setting up a CoC

**Comment by amangoel185** [comment]
19 October 2022 (Wednesday): @amangoel185

* Discussion with Saranjeet about the requirements of the CoC
* Collecting and browsing CoCs from other communities to get a better idea of what we want

**Comment by jyoti-bhogal** [comment]
19 October 2022 (Wednesday) @jyoti-bhogal 
1. Made progress on writing the draft of blog
2. Progressed with creating one PR https://github.com/rse-asia/RSE_Asia/pull/45
3. Progressed on integrating my GitHub with RStudio with help from @SaranjeetKaur

**Comment by kozo2** [comment]
21 October 2022 (Friday) @kozo2 
1. Asked @joelnitta to review the Japanese post PR #32 
2. Still translating #32 to English at #44

**Comment by jyoti-bhogal** [comment]
21 October 2022 (Friday) @jyoti-bhogal 
1. Made changes to My OLS-5 Experience Blog
2. Created 3 PRs for the same for Hactoberfest
3. Learnt to commit and push changes from RStudio Desktop to GitHub Repository with help from @SaranjeetKaur

**Comment by SaranjeetKaur** [comment]
21 October 2022 (Friday) @SaranjeetKaur 
- Assisted @jyoti-bhogal in pushing her commits to the repo
- Added html for OLS 5 blog to the repo
- Reached out to the RSE community, the Turing Way community, and the Carpentries community for a Reviewer for Kozo's blog

**Comment by SaranjeetKaur** [comment]
25 October 2022 (Tuesday) @SaranjeetKaur 

Attended some talks for the International Open Access Week 2022

**Comment by SaranjeetKaur** [comment]
26 October 2022 (Wednesday) @SaranjeetKaur 

- Discussed the blogs' status with both @jyoti-bhogal and @kozo2 
- Discussed about the CoC with @sayantikabanik
    - Whether to have a generic CoC or a specific one (each for contirbutor, community, events, etc)
    - Compare with other existing and relevant CoCs (like other RSE organisations and different communities like PyLadies, FOSS Asia, International Council of RSEs, NumFOCUS, etc.)
    - Have a separate page on the website for the CoC
    - Acknowledge the communities (on the above page) whose CoC we are taking inspiration from

**Reply by SaranjeetKaur** [reply]
@amangoel185 this might be relevant for your work too

**Reply by sayantikabanik** [reply]
#### More thoughts and potential way forward

- CoC committee should be operating centrally, being transparent and assessing each potential violation without bias.
- Having a geography-specific CoC might not be required. Based on my experience global communities follow one single code of conduct.
- Given we might not have clarity on all the aspects related to global RSE operations. Having a discussion around the same could be helpful.
- Potential plan of action to have an accessible and secure questionnaire in a *form* to report the CoC violations could be a good addition. Inspiration taken from [NumFOCUS](https://numfocus.org/code-of-conduct).

**Comment by amangoel185** [comment]
27 October 2022 (Thursday): @amangoel185

- Reviewed notes and discussion on CoC
- Added all the relevant information to #13
- Reading CoCs from other communities

**Comment by kozo2** [comment]
27 October 2022 (Thursday): @kozo2 
- Updated #44
